index.js

```diff
@@ -270,29 +281,39 @@ class CoverageSonar extends CoverageBase {
      * Return links to the Sonar project and coverage metadata
      * @method getInfo
      * @param   {Object}  config
-     * @param   {String}  config.jobId          Screwdriver job ID
-     * @param   {String}  [config.pipelineId]   Screwdriver pipeline ID (if enterprise is enabled)
-     * @param   {String}  [config.prNum]        Pull request number
-     * @param   {String}  config.startTime      Job start time
-     * @param   {String}  config.endTime        Job end time
-     * @return  {Promise}                       An object with:
-     *                                          - tests success percentage
-     *                                          - coverage percentage
-     *                                          - project url
-     *                                          - Sonar env vars
+     * @param   {Object}  [config.annotations]      Screwdriver job annotations
+     * @param   {String}  config.jobId              Screwdriver job ID
+     * @param   {String}  config.jobName            Screwdriver job name
+     * @param   {String}  config.pipelineId         Screwdriver pipeline ID (if enterprise is enabled)
+     * @param   {String}  config.pipelineName       Screwdriver pipeline name
+     * @param   {String}  [config.prNum]            Pull request number
+     * @param   {String}  config.startTime          Job start time
+     * @param   {String}  config.endTime            Job end time
+     * @param   {String}  [config.sonarProjectKey]  Sonar project key
```

Comment:
seems like this is called `coverageProjectKey` ?

https://github.com/screwdriver-cd/coverage-base/pull/12/files#diff-04c6e90faac2675aa89e2176d2eec7d8R39

https://github.com/screwdriver-cd/screwdriver/pull/2161/files#diff-b2d3b15fbf69271cc115dba8502fc218R29